mteb/tasks/Classification/ces/CSFDCZMovieReviewSentimentClassification.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 2048
+
+
+class CSFDCZMovieReviewSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="CSFDCZMovieReviewSentimentClassification",
+        description="The dataset contains 30k user reviews from csfd.cz in Czech.",
+        reference="https://arxiv.org/abs/2304.01922",
+        dataset={
+            "path": "fewshot-goes-multilingual/cs_csfd-movie-reviews",
+            "revision": "dd2ede6faaea338ef6b1e2966f06808656975a23",
+        },
+        type="Classification",
+        category="s2s",
+        date=("2002-06-28", "2020-03-13"),
+        eval_splits=["test"],
+        eval_langs=["ces-Latn"],
+        main_score="accuracy",
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="CC-BY-SA-4.0",
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+@misc{štefánik2023resources,
+      title={Resources and Few-shot Learners for In-context Learning in Slavic Languages}, 
```

Comment:
Is this the correct citation?
The paper doesn't seem to mention this dataset

Reply:
@KranthiGV It actually does in Section 3.1.1 (page 3 of https://arxiv.org/pdf/2304.01922):

![image](https://github.com/embeddings-benchmark/mteb/assets/461491/f00adfeb-3cf6-400e-ad38-8d176052dc28)